update_image

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+#This script create/update image by getting the the parameter of os release pair splitted with comma
+# Sample platform names 'debian,buster', 'centos,7' etc.
+
+# make bash behave
+set -euo pipefail
+IFS=$'\n\t'
+
+pgversions='10 11 12 13 14'
+topdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+dockerfiles_dir="${topdir}/dockerfiles"
+
+badusage=64
+
+
+nprocs="${1:-1}"
+
+declare args
+
+
+IFS=',' read -r os release <<< "${TARGET_PLATFORM}"
```

Comment:
Can we at least loop through all supported platform names when ${TARGET_PLATFORM} is empty?

This way, we will be able to update all images locally and have a more robust development environment.

Reply:
That is why I lost lots of time. I didn't notice that getting the full list. It tried to execute all the images.
I can write another script for that. 
https://github.com/citusdata/packaging/issues/830